jobs/jobs.yml

```diff
@@ -178,9 +178,19 @@
         - 'plone-{plone-version}-python-{py}-i18n-find-missing'
 
 - project:
-    name: Translations of Volto
+    name: Translations of Volto (16.x.x)
+    branch:
+        - '16.x.x'
     jobs:
-        - 'volto-translations'
+        - 'volto-translations-{branch}'
+
+- project:
+    name: Translations of Volto (master)
+    branch:
+        - 'master'
+    jobs:
+        - 'volto-translations-{branch}'
+
```

Comment:
You don't need to create 2 `project:` keys, something like this is enough:

```yml
project:
    name: Translations Volto
    branch:
        - '16.x.x'
        - 'master'
    jobs:
        - 'volto-translations-{branch}'
```
🍀